packages/scripts/src/local/getHistoricalPrice.js

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env node
+
+/**
+ * @notice This is an example script demonstrating how to get a historical price at a specific timestamp.
+ * It could be used for example to query a price before committing a vote for a DVM price request.
+ * We provide default configurations for querying median prices on specific markets across some exchanges and markets.
+ * This script should serve as a template for constructing other historical median queries.
+ *
+ * @notice This script will fail if the `--time` is not within a the caller's configured lookback window. The caller
+ * set this via the PRICEFEED_CONFIG={"lookback":x} environment variable.
+ * @dev How to run:
+ *     HARDHAT_NETWORK=mainnet ./src/local/getHistoricalPrice.js
```

Comment:
nit: don't you need to run this command with with `node x`? Or can you just run the `.js` file directly?

Reply:
I put the shebang at the top so you don't need it.